.github/workflows/vcpkg_docker.yml

```diff
@@ -0,0 +1,137 @@
+name: Docker Build Images
+
+# Based on https://docs.docker.com/build/ci/github-actions/multi-platform/#distribute-build-across-multiple-runners
+
+on:
+  schedule:
+    # Once every Wednesday at 00:00
+    - cron: '0 0 * * 3'
+  push:
+    branches:
+      - master
+    paths:
+      - 'docker/**'
+      - '.github/workflows/vcpkg_docker.yml'
+  pull_request:
+    paths:
+      - 'docker/**'
+      - '.github/workflows/vcpkg_docker.yml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      # This needs to be the same as in the `merge` job
+      # Also remember to change the 'docker/build.sh' script
+      REGISTRY_IMAGE: ghcr.io/lifting-bits/cxx-common/vcpkg-builder-ubuntu-${{ matrix.ubuntu_version }}
```

Comment:
might be better to extra the parts that are the same to a top level environment variable and then in here, just append the matrix.ubuntu_version so we only have to change one spot if we want to change the prefix

Reply:
I thought of this, but you're not allowed to use `env` variables in another `env` block 😞